python_modules/dagster/dagster/_check/__init__.py

```diff
@@ -882,6 +882,31 @@ def is_list(
     return _check_iterable_items(obj, of_type, "list")
 
 
+# ########################
+# ##### LITERAL
+# ########################
+
+
+def literal_param(
+    obj: T, param_name: str, values: Sequence[object], additional_message: Optional[str] = None
+) -> T:
+    additional_message = " " + additional_message if additional_message else ""
```

Comment:
nit: could move this inside of conditional or delete and rely on `_param_value_mismatch_exception`  [1]

Reply:
oh yeah whoops that was left over from before I factored out `_param_value_mismatch_exception`